### DISCUSSION TITLE
Winget will install updates that WingetUI gets stuck updating

### DISCUSSION THREAD

**Discussion by Aston77** [Devolutions/UniGetUI#1048] [Q&A] [answered]
I keep getting notifications of two updates ready to install (the 2nd one looks strange though). The problem is the install just loops endlessly.

![Screenshot 2023-05-22 090005](https://github.com/marticliment/WingetUI/assets/78501124/24e80e57-7c6e-4486-a39e-92bff7b6ee0b)

I was able to run Winget separately and install the Discord update while WingetUI was still looping.

![image](https://github.com/marticliment/WingetUI/assets/78501124/4831a635-3fc4-4c0a-b712-e9ce14a090d7)

Is there a setting I could have improperly configured that would result in behavior like this?

**Comment by marticliment** [comment]
Try blacklisting them

**Reply by Aston77** [reply, marked as answer]
Before (having issues)
![image](https://github.com/marticliment/WingetUI/assets/78501124/2d83a94a-6e16-40ec-a63d-761cd7da8b47)
I cross-referenced with another install I had that was working fine and as soon as I unchecked all of the administrator preferences the issue resolved.

After (working)
![image](https://github.com/marticliment/WingetUI/assets/78501124/9f6ad3fa-0e02-4d77-a8eb-85d55aa7b45e)
